checkin/models.py

```diff
@@ -29,6 +52,13 @@ def __str__(self):
 
 
 class UserSkill(models.Model):
+    """
+    Model for user's skill, storing skill and skill level of skill
+
+    :var profile: Profile of the user which this skill belongs to
+    :var skill: skill model
+    :var skill_level: Three levels, 1=beginner, 3=expert
```

Comment:
The user's skill level, can be one of three levels: "beginner", "intermediate" or "expert"